m3u8/parser.py

```diff
@@ -15,7 +15,6 @@
 http://stackoverflow.com/questions/2785755/how-to-split-but-ignore-separators-in-quoted-strings-in-python
 '''
 ATTRIBUTELISTPATTERN = re.compile(r'''((?:[^,"']|"[^"]*"|'[^']*')+)''')
-URI_PREFIXES = ('https://', 'http://', 's3://', 's3a://', 's3n://')
```

Comment:
This change allows playlists to use additional URL schemes without having to update this list. [RFC 8216](https://datatracker.ietf.org/doc/html/draft-pantos-hls-rfc8216bis#section-4.4.6.6) does mention at least the `data://` scheme.

Reply:
Awesome! You remove a line while adding a new feature 👏🏼